interceptors/src/unit-test/java/uk/gov/moj/sdt/interceptors/in/ServiceRequestInboundInterceptorTest.java

```diff
@@ -1,173 +1,175 @@
- * disclaimed. In no event shall the Ministry of Justice or its contributors be liable for any
- * direct, indirect, incidental, special, exemplary, or consequential damages (including, but
- * not limited to, procurement of substitute goods or services; loss of use, data, or profits;
- * or business interruption). However caused any on any theory of liability, whether in contract,
- * strict liability, or tort (including negligence or otherwise) arising in any way out of the use of this
- * software, even if advised of the possibility of such damage.
- *
- * $Id: $
- * $LastChangedRevision: $
- * $LastChangedDate: $
- * $LastChangedBy: $ */
-package uk.gov.moj.sdt.interceptors.in;
-
-import java.io.BufferedReader;
-import java.io.IOException;
-import java.io.InputStream;
-import java.io.InputStreamReader;
-
-import org.apache.cxf.binding.soap.SoapMessage;
-import org.apache.cxf.message.MessageImpl;
-import org.easymock.EasyMock;
-import org.junit.Assert;
-import org.junit.Test;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-import uk.gov.moj.sdt.dao.GenericDao;
-import uk.gov.moj.sdt.domain.ServiceRequest;
-import uk.gov.moj.sdt.utils.AbstractSdtUnitTestBase;
-import uk.gov.moj.sdt.utils.SdtContext;
-
-/**
- * Test class.
- *
- * @author d195274
- */
-public class ServiceRequestInboundInterceptorTest extends AbstractSdtUnitTestBase {
-    /**
-     * Logger object.
-     */
-    private static final Logger LOGGER = LoggerFactory.getLogger(ServiceRequestInboundInterceptorTest.class);
-
-    /**
-     * Build a mocked dao.
-     *
-     * @param serviceRequest possible superfluous object.
-     * @return the mocked dao.
-     */
-    private GenericDao getMockedGenericDao(final ServiceRequest serviceRequest) {
-        final GenericDao mockServiceRequestDao = EasyMock.createNiceMock(GenericDao.class);
-        mockServiceRequestDao.persist(serviceRequest);
-        EasyMock.expectLastCall();
-        EasyMock.replay(mockServiceRequestDao);
-        return mockServiceRequestDao;
-    }
-
-    /**
-     * Test that the process correctly works via mocked out extensions.
-     */
-    @Test
-    public void testHandleMessage() {
-        try {
-            // Create the service request inbound interceptor.
-            final ServiceRequestInboundInterceptor sRII = new ServiceRequestInboundInterceptor();
-
-            // Inject dummy service request into interceptor.
-            sRII.setServiceRequestDao(getMockedGenericDao(new ServiceRequest()));
-
-            // Setup the raw XML as if the XmlInboundInterceptor had run.
-            final String xml =
-                    this.convertStreamToString(ServiceRequestInboundInterceptorTest.class
-                            .getResourceAsStream("inRequest.xml"));
-            SdtContext.getContext().setRawInXml(xml);
-
-            // Call the code to be tested.
-            sRII.handleMessage(new SoapMessage(new MessageImpl()));

```

Comment:
Curious. But not this PR's problem...

Reply:
nothing changed in this class, except adding below code at line number 83
 ServiceRequestDao serviceRequestDao = EasyMock.mock(ServiceRequestDao.class);